src/assets/js/chameleon.js

```diff
@@ -189,11 +189,11 @@ function PushValuesToURL( inputs ) {
  * @param {array}  customInputs - The inputs to select the color values from
  */
 function ApplyColorsToColorSquare( customInputs ) {
```

Comment:
You renamed the function but the docs doesn't match it. Make sure you update the docs above.